music21/musicxml/xmlToM21.py

```diff
@@ -2318,13 +2312,13 @@ def parse(self):
 
         if self.useVoices is True:
             for v in self.stream.iter.voices:
-                if v: # do not bother with empty voices
+                if v:  # do not bother with empty voices
                     v.makeRests(inPlace=True, hideRests=True)
                     v.coreElementsChanged()
         self.stream.coreElementsChanged()
 
         if (self.restAndNoteCount['rest'] == 1
-                and self.restAndNoteCount['note'] == 0):
```

Comment:
the reindentation here will set off my lint-checker as it makes a continuation of the if look the same level as the unindented lines.